algorithms/appo/learner.py

```diff
@@ -399,6 +409,26 @@ def _value_loss(self, new_values, old_values, target, clip_value):
 
         return value_loss
 
+    def entropy_exploration_loss(self, action_distribution):
+        entropy = action_distribution.entropy()
+        if self.cfg.exploration_loss_coeff > 0.0:
+            entropy_loss = -self.cfg.exploration_loss_coeff * entropy.mean()
+        else:
+            entropy_loss = 0.0
+        return entropy_loss
+
+    def symmetric_kl_exploration_loss(self, action_distribution):
+        if self.cfg.exploration_loss_coeff > 0.0:
+            kl_prior = action_distribution.symmetric_kl_with_uniform_prior()
+            if not torch.isfinite(kl_prior).all():
+                kl_prior = torch.zeros(kl_prior.shape)
```

Comment:
add log.error here, this should be as visible as possible

Reply:
actually you can check this after you do `.mean()`, it should be much faster. If one element is infinite then mean will be infinite as well. Also check for NaN

Reply:
the `isfinite()` function checks for both infinity and nan